### PR TITLE
add `\[` and `\]` to PS3

### PR DESCRIPTION
**Pull request by retokromer**
@dericed Could you possibly check this on your side? Should resolve https://github.com/amiaopensource/ltopers/issues/63 in a more elegant way.

### CODE REVIEW

File: formatlto [hunk 1]
```diff
@@ -56,7 +54,7 @@ if [ ! "${LTO_ARRAY}" ] ; then
 fi
 
 if [[ "${#LTO_ARRAY[@]}" -gt 1 ]] ; then
-   PS3="${BLUE}Which LTO deck?${NC} "
+   PS3='\[\e[1;34m\]Which LTO deck? \[\e[0m\]'
```
Comment (dericed) [inline formatlto]: Still outputs literally for me on macOS.

```
\[\e[1;34m\]Which LTO deck? \[\e[0m\]
```

Comment (retokromer) [inline formatlto]: That’s strange! Works fine on my side, both on Linux and on macOS, but in this case of course I go back to the ugly version.

Comment (dericed) [inline formatlto]: retested to examine if an user error, but I had the same result.

Comment (retokromer) [inline formatlto]: Thank you again!

The only difference I know is that you use LTFS from Quantum and I from HP, but that does not affect Bash. I’m working with Bash 4.4.12.

**Comment by dericed** [comment]
```
bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin16)
Copyright (C) 2007 Free Software Foundation, Inc.
```

File: formatlto (hunk 1 above)
Comment (dericed) [inline formatlto]:
```
bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin16)
Copyright (C) 2007 Free Software Foundation, Inc.
```

Comment (retokromer) [inline formatlto]: Thanks! I’ll check if something with `PS1`, …, `PS4` has changed from 3 to 4.

On my side:
```
GNU bash, Version 4.4.12(1)-release (x86_64-apple-darwin15.6.0)
Copyright (C) 2016 Free Software Foundation, Inc.
```

Comment (retokromer) [inline formatlto]: It’s not related to the Bash version, but to our main configuration. Usually **PS3 is not escaped**, therefore we need indeed to use `tput` instead here. Sorry for boring you with code elegance… ;-)

**Pull request closed without merging**